checks/permissions.go

```diff
@@ -231,13 +246,21 @@ func calculateScore(result permissionCbData) int {
 	// Note: there are legitimate reasons to use some of the permissions like checks, deployments, etc.
 	// in CI/CD systems https://docs.travis-ci.com/user/github-oauth-scopes/.
 
-	if permissionIsPresent(result, "all") {
-		return checker.MinResultScore
-	}
-
 	// Start with a perfect score.
 	score := float32(checker.MaxResultScore)
 
+	// If no top level permissions are defined...
+	if permissionIsPresentInTopLevel(result, "all") {
+		switch permissionIsPresentInRunLevel(result, "all") {
```

Comment:
Any particular reason why you're using a switch here? A simple if-else seems more readable.

Reply:
nop. I have changed to an if statement. Thanks